<launch>
    <!-- Enable rviz visualization -->
    <arg name="rviz"        default="true" />
    <!-- Set nodes on debug mode -->
    <arg name="debug"       default="false" />

    <!-- The task file for the stoc. -->
    <arg name="stocFile"    value="$(find ocs2_stoc_mobile_manipulator)/config/mabi_mobile/stoc.info" />
    <!-- The URDF model of the robot -->
    <arg name="urdfFile"    value="$(find ocs2_robotic_assets)/resources/mobile_manipulator/mabi_mobile/urdf/mabi_mobile.urdf" />
    <!-- The task file for the mpc. -->
    <arg name="taskFile"    value="$(find ocs2_mobile_manipulator)/config/mabi_mobile/task.info" />
    <!-- The library folder to generate CppAD codegen into -->
    <arg name="libFolder"   value="$(find ocs2_stoc_mobile_manipulator)/auto_generated/mabi_mobile" />

    <include file="$(find ocs2_stoc_mobile_manipulator_ros)/launch/include/mobile_manipulator.launch">
        <arg name="rviz"      value="$(arg rviz)" />
        <arg name="debug"     value="$(arg debug)" />
        <arg name="stocFile"    value="$(arg stocFile)" />
        <arg name="urdfFile"  value="$(arg urdfFile)" />
        <arg name="taskFile"  value="$(arg taskFile)" />
        <arg name="libFolder" value="$(arg libFolder)" />
    </include>
</launch>